bot/feedback/feedback.py

```diff
@@ -0,0 +1,303 @@
+                self._db_connector.get_suggestion_status(payload.message_id) == SuggestionStatus.UNDECIDED:
+            message = await self.bot.get_channel(payload.channel_id).fetch_message(payload.message_id)
+            reactions = message.reactions
+
+            if payload.emoji.name in (constants.EMOJI_UPVOTE, constants.EMOJI_DOWNVOTE):
+                total_votes = reactions[0].count + reactions[1].count
+                difference = abs(reactions[0].count - reactions[1].count)
+
+                # Changes the color of the embed depending if one side received at least 10 votes and the difference
+                # between them is bigger than 50% of total votes.
+                if constants.LIMIT_SUGGESTION_VOTES < reactions[0].count > reactions[1].count and \
+                        difference > (total_votes / 2):
+                    new_embed = _recolor_embed(message.embeds[0], constants.EMBED_COLOR_SUGGESTION_MEMBERS_LIKE)
+                    await message.edit(embed=new_embed)
+                elif constants.LIMIT_SUGGESTION_VOTES < reactions[1].count and difference > (total_votes / 2):
+                    new_embed = _recolor_embed(message.embeds[0], constants.EMBED_COLOR_SUGGESTION_MEMBERS_DISLIKE)
+                    await message.edit(embed=new_embed)
+
+
+async def _notify_user_suggestion_change(guild: discord.Guild, user_id: int):
+    """Method which notifies a user about changes regarding his suggestion.
+
+    Gets the corresponding Discord member and sends a personalized DM to him informing him about changes regarding his
+    suggestion.
+
+    Args:
+        guild (discord.Guild): The Discord server on which to find the suggestion.
+        user_id (int): The id of the user who submitted the suggestion.
+    """
+    member = guild.get_member(user_id)
+    name = member.nick if member.nick else member.name
+    text = "Hey, {0}!\nEs gibt Neuigkeiten bezüglich deines Vorschlags.Sieh am besten gleich in <#{1}> nach, wie " \
+           "das Urteil ausgefallen ist. :fingers_crossed:".format(name, constants.CHANNEL_ID_SUGGESTIONS)
+
+    await member.send(text)
+
+
+async def _refresh_suggestion_embed(message: discord.Message, author: discord.Member, reason: Optional[str],
+                                    status: SuggestionStatus):
+    """Method which adapts an embed regarding a suggestion.
+
+    Changes title, color and fields of the embed accordingly to the new status of the suggestion.
+
+    Args:
+        message (discord.Message): The message containing the suggestion embed.
+        author (discord.Member): The user who invoked the command to change it.
+        reason (Optional[str]): The reason for the decision.
+        status (SuggestionStatus): The new status of the suggestion.
+    """
+    dict_embed = message.embeds[0].to_dict()
+
+    if "fields" in dict_embed:
+        dict_embed["fields"].clear()
+    else:
+        dict_embed["fields"] = list()
```

Comment:
Why do you need to make a distinction and not just always replace it with an empty list?

Reply:
Good point. Idk, maybe I was tired.